src/components/vm/vmActions.jsx

```diff
@@ -467,6 +470,17 @@ const VmActions = ({ vm, vms, onAddErrorNotification, isDetailsPage }) => {
     if (domainCanRename(state) || isDetailsPage)
         dropdownItems.push(<Divider key="separator-rename" />);
 
+    if (!vm.hasTPM) {
```

Comment:
Should we check `vm.capabilities.supportsTPM` here as well?

Reply:
Yes, good idea -- works, done.